src/test/App.test.js

```diff
@@ -1,3 +1,14 @@
 import React from 'react'
 import { MemoryRouter } from 'react-router-dom'
 import App from '../App'
+
+it('App renders correctly', () => {
+  const tree = renderer
+    .create(
+      <MemoryRouter>
+        <App />
+      </MemoryRouter>
+    )
+    .toJSON();
+  expect(tree).toMatchSnapshot();
+})
```

Comment:
- [ ] Good job so far 👍 but one of your tests is failing. Kindly fix this 💯 
- Your site:
![Annotation 2022-06-09 060827](https://user-images.githubusercontent.com/85113567/172769020-69037bcc-6d79-4bee-b51e-8a6e75943cbb.jpg)